main-window.js

```diff
@@ -65,7 +67,15 @@ module.exports = function (config) {
     electron.remote.app.setBadgeCount(count)
   })
 
-  insertCss(require('./styles'))
+
+  watch(api.settings.obs.get('patchwork.theme', 'light'), name => {
+    Array.from(document.head.children)
```

Comment:
is this to reset the previous styles? i reckon would make more sense to have a single `<style>` element with an id, then override just the element. is easy with https://github.com/bendrucker/insert-styles

Reply:
good to know about that. I need to keep building and polishing other things. 

I'm really feeling some of the provocation of that talk "worse is better" at the moment - just make it bad, and work on the important things. For me, the important part here was enabling Alan, and making this tidy enough for people to experiment with. It could be a bad idea, or it could be something which should be branched out in the future, I don't know. 

If you'd like to improve it, I'm totally happy for others to change / adapt what I've done here.

If you think this is too light an involvement to be net good, would love to hear :heart: